components/cli/pkg/commands/image/test.go

```diff
@@ -41,7 +42,7 @@ import (
 	"github.com/cellery-io/sdk/components/cli/pkg/version"
 )
 
-// RunRun starts Cell instance (along with dependency instances if specified by the user)\
+// RunTest starts Cell instance (along with dependency instances if specified by the user)\
 func RunTest(cli cli.Cli, cellImageTag string, instanceName string, startDependencies bool, shareDependencies bool,
 	dependencyLinks []string, envVars []string, assumeYes bool, debug bool, verbose bool, incell bool, projLocation string) error {
 	spinner := util.StartNewSpinner("Extracting Cell Image " + util.Bold(cellImageTag))
```

Comment:
Do not directly use  the spinner. Use the ExecuteTask interface function of cli instead. Please refer run.go.

Reply:
Fixed